analyzer/src/main/kotlin/managers/GoMod.kt

```diff
@@ -390,6 +362,18 @@ private const val WHY_CHUNK_SIZE = 32
 
 private val JSON: Json by lazy { Json { ignoreUnknownKeys = true } }
 
+private const val DEFAULT_GO_PROXY = "https://proxy.golang.org"
```

Comment:
Nit: This also moves this property (not only functions) and makes it private. For completeness, that should be mentioned as part of the commit message, IMO.